composer.json

```diff
@@ -52,13 +52,17 @@
     "symfony/phpunit-bridge": "^4.4"
   },
   "scripts": {
-    "post-install-cmd": [
-      "SuplaBundle\\DependencyInjection\\GitVersionDumper::dumpVersion"
-    ],
-    "post-update-cmd": [
-      "SuplaBundle\\DependencyInjection\\GitVersionDumper::dumpVersion"
-    ],
-    "test": [
+    "dump-version" : "SuplaBundle\\DependencyInjection\\GitVersionDumper::dumpVersion",
+		    "build-parameters" : "Incenteev\\ParameterHandler\\ScriptHandler::buildParameters",
+    		"post-install-cmd" : [
```

Comment:
straszny tutaj bałagan z wcięciami się zrobił. Używaj proszę spacji do wcięć

Reply:
Porawiłem

Reply:
Nie do końca, bo tutaj jest wcięcie mniejsze, a linię niżej większe, a to ten sam poziom JSONa